PlayerCollection.cs

```diff
@@ -218,5 +256,19 @@ private Dropdown AddItemToDropddown(Dropdown dropdown, string newItem)
             return dropdown;
         }
 
+        private string GetPreviousCharactersToolTipText()
+        {
+            var tooltip = "Previously played...\r\n";
+            var last = _player.PreviouslyPlayedCharacters.Last();
+            foreach (var character in _player.PreviouslyPlayedCharacters)
+            {
+                tooltip += $"{Specialization.GetEliteName(character.CurrentSpecialization, character.Profession)} ({character.CharacterName})";
+                if (character.CharacterName != last.CharacterName)
+                {
+                    tooltip += "\r\n";
+                }
+            }
+            return tooltip;
```

Comment:
suggestion: using a [StringBuilder](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder) and its methods [AppendLine](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendline?view=net-6.0#system-text-stringbuilder-appendline(system-string)) or [AppendJoin](https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendjoin?view=net-6.0#system-text-stringbuilder-appendjoin-1(system-string-system-collections-generic-ienumerable((-0)))) may make the code more straightforward!